orisa/orisa.py

```diff
@@ -1402,14 +1402,14 @@ def _create_help(self, ctx):
             name="!ow config",
             # Translators: don't translate "Orisa Admin"
             value=_(
-                'This command can only be used by members with the "Orisa Admin" role and allows them to configure Orisa for the specific Discord server.'
+                "This command can only be used by members with the \"Orisa Admin\" role and allows them to configure Orisa for the specific Discord server."
             ),
             inline=False,
         )
         embed.add_field(
             name="!ow dumpsr",
             # Translators: help for !ow dumpsr
-            value=_("Download your SR history as an Excel sheet"),
```

Comment:
Spreadsheet is ambiguous. Is is LibreOffice .ods, or some other format? I wanted to be explicit that it is in Excel format.

Reply:
Actually Excel is ambiguous, since it, as LibreOffice, Abiword, and Apache OpenOffice etc. supports many different formats. All suites use different formats for different things. What format is it specifically?

Reply:
I've gone for Excel spreadsheet

Reply:
There is no reason to specify the editor, they all have support of all ISO standards.
Which makes it all the more strange to consider variants MS has made. People left the ISO in protest when those standards passed.